apps/game_backend/priv/repo/migrations/20240425171037_add_arena_match_results.exs

```diff
@@ -3,7 +3,7 @@ defmodule GameBackend.Repo.Migrations.AddArenaMatchResults do
 
   def change do
     create table(:arena_match_results) do
-      add :user_id, references(:google_users, on_delete: :delete_all), null: false
+      add :google_user_id, references(:google_users, on_delete: :delete_all), null: false
```

Comment:
Whoops, yeah this was something i added to tests this and was left here 😂  created his proper migration changing this! also i wonder if is worth to rename this or change it to belong to a user instead of a google user, what do you think?